src/jasmine.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env jasmine */
+import { assertions } from 'redux-actions-assertions-js';
+
+const toDispatchActions = () => {
```

Comment:
Please use functions declarations as it is done in other files to keep format consistency